kivy/graphics/transformation.pyx

```diff
@@ -611,6 +611,20 @@ cdef class Matrix:
 
         return (winx, winy, winz)
 
+    def transform(self, list point):
```

Comment:
I'd suggest to not type it as list, because what if point is a tuple? Or event better, type it as `double x, double y`.